### ISSUE TITLE
Add build context of solution

### ISSUE THREAD

**Issue by hulucc** [Serg046/MsBuildTaskExplorer#22]
The solution build context is undefinded now.
And the MsBuildTaskExplorer has the knowledge of which solution is used.
```
SolutionDir = *Undefined*
SolutionExt = *Undefined*
SolutionFileName = *Undefined*
SolutionName = *Undefined*
SolutionPath = *Undefined*
```

**Comment by Serg046** [comment]
Soon

**Comment by Serg046** [comment]
Done and released to version 1.3.1
But do not forget that using solution variables is not best practice because the project's target can be executed directly outside of solution, e.g. by CI build.
